lib/ReactViews/FeatureInfo/FeatureInfoPanel.jsx

```diff
@@ -101,6 +101,21 @@ const FeatureInfoPanel = React.createClass({
         }
     },
 
+    getMessageForNoResults() {
+        if(this.props.terria.nowViewing.hasItems) {
+            // if no local storage flag
```

Comment:
Sorry to be pedantic - these comments are out-of-date though :-)
And to be even more uselessly pedantic - I think everywhere else we're writing `if (...)` not `if(...)`... do you mind adding in the space here and in `DataCatalogItem` above?